app/controllers/contact_type_groups_controller.rb

```diff
@@ -0,0 +1,25 @@
+class ContactTypeGroupsController < ApplicationController
+  before_action :must_be_admin
+
+  def new
+    @contact_type_group = ContactTypeGroup.new
+  end
+
+  def create
+    @contact_type_group = ContactTypeGroup.new(contact_type_group_params.merge(casa_org: current_organization))
+
+    respond_to do |format|
```

Comment:
Do we need the `respond_to` block even though we are dealing with `HTML` only?

Reply:
Whoops, left over.